client/my-sites/site-settings/composing/markdown.jsx

```diff
@@ -0,0 +1,40 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FormFieldset from 'components/forms/form-fieldset';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import ExternalLink from 'components/external-link';
+import JetpackModuleToggle from '../jetpack-module-toggle';
+
+const Markdown = ( { disabled, siteId, translate } ) => (
+	<FormFieldset className="composing__markdown has-divider is-top-only">
+		<JetpackModuleToggle
+			disabled={ disabled }
+			label={ translate( 'Write posts or pages in plain-text Markdown syntax' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **11** times:
`translate( 'Write posts or pages in plain text Markdown syntax.' )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).